fifo_cache.py

```
@@ -0,0 +1,28 @@
+from collections import OrderedDict
+
+class FIFOCache:
+    """
+    FIFO (First-In First-Out) 缓存实现
+
+    先进先出，最早插入的最早淘汰。
+    """
+    def __init__(self, capacity: int):
+        self.capacity = capacity
+        self.cache = dict()  # 问题1：应为 OrderedDict，dict无序
```

Ruff (0.14.6)

Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)